src/functions-and-arrays.js

```diff
@@ -52,15 +110,40 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray (arrayWithDublicatesEntries) {
+
+  if (arrayWithDublicatesEntries.length === 0) 
+    return null;
+
+    let arrayWithDublicates = []; 
+
+  for (let entry of arrayWithDublicatesEntries) { 
+  if (!arrayWithDublicatesEntries.includes(entry))
+     {arrayWithDublicatesEntries.push(entry)}
```

Comment:
Instead of `arrayWithDublicatesEntries`, you should work here with the `arrayWithoutDuplicates`